### ISSUE TITLE
Facet / Filter selection changes should announce to screen reader

### ISSUE THREAD

**Issue by zebdelk** [cedardevs/onestop#370]
When selecting a facet, screen readers have no way to know that an updated search was triggered. At best, they might get an announcement that search results have loaded.

The correct behavior should be that when a user selects a facet, the screen reader informs the user that a search has been made, and again when the results have loaded.

**Comment by ajakz** [comment]
👍
